forkable/forkdb.go

```diff
@@ -67,14 +67,15 @@ func (f *ForkDB) SetName(name string) {
 // unknown behaviour if it was already set ... maybe it explodes
 func (f *ForkDB) TrySetLIB(headRef, previousRef bstream.BlockRef, libNum uint64) {
 
-	// special case for EOS: block 2 is always good
-	if headRef.Num() == 2 && previousRef.Num() == 1 {
+	//if headRef.Num() == bstream.GetProtocolFirstStreamableBlock && previousRef.Num() == bstream.GetProtocolGenesisBlock {
```

Comment:
Can you clean up commented code and fix me comment, the fix me don't belongs here.